### ISSUE TITLE
macos does not accept unquoted strings in repl

### ISSUE THREAD

**Issue by jnc-nj** [vindarel/replic#9]
## System
macOS High Sierra 10.13.6
roswell 19.06.10.100(NO-GIT-REVISION)
build with Apple LLVM version 10.0.0 (clang-1000.11.45.5)
libcurl=7.54.0
Quicklisp=2019-02-16
Dist=2019-07-11
sbcl-bin/1.5.5
sbcl/1.5.5

## Problem
repl does not accept commands without quotes on this system (macos)
reported failures also on dockerized repls (ubuntu 16.04/ubuntu 18.04)

## Example
`ros build repl.ros`
`./repl -a 127.0.0.1 -p 9000`
`connect server 127.0.0.1:9001` --> `Error: invalid number of arguments: 1`
`connect "server" "127.0.0.1:9001"` --> outputs properly
weird behaviour with single argument functions
`ls fns` -->
```
" fns"
  [simple-string]

Element-type: CHARACTER
Length: 4
```
where `ls` is a function which takes an argument which is the key of a hash-table, and return the value

**Comment by vindarel** [comment]
I might have a clue, I'll have a look shortly©. Maybe is it a regression from 0.11. You could try with version 0.10 (cloning the project and stepping at the right commit, with Qlot,…).

Can you still show the `connect` function ? 

Can you show a function that works properly ?

**Comment by vindarel** [comment]
Indeed, it seems that the function args are badly parsed here: https://github.com/vindarel/replic/blob/master/src/replic.lisp#L297

**Comment by vindarel** [comment]
It should be fixed, you confirm?

**Comment by jnc-nj** [comment]
> It should be fixed, you confirm?

Sorry I was on holiday, moon festival in China.
We've got a new build coming up on Wednesday, i'll do the tests for each build then and let you know if it occurs again.

**Comment by jnc-nj** [comment]
> It should be fixed, you confirm?

Yes I can confirm it has been fixed on ubuntu 16.04/18.04 and macos.
Thank you!

**Comment by vindarel** [comment]
Great, thanks!
